containers/cp-test

```diff
@@ -357,79 +180,15 @@ if [ -f /usr/bin/supervisord ]; then
     /usr/bin/supervisord -c /etc/supervisord.conf
 fi
 
-# Make sure we update the ruby bundle:
-if [ ! -f ./gradlew ]; then
-    bundle install --without=proton
-fi
-
 mkdir -p /var/log/candlepin
 
-# Setup our tee pipe
-mkfifo /tmp/teepipe
-
-if [ "$LINTER" == "1" ]; then
-    echo "Running linter..."
-    CLEAN_CP=1
-
-    cd $CP_HOME
-    tee /var/log/candlepin/lint.log < /tmp/teepipe &
-    build_lint > /tmp/teepipe 2>&1
-fi
-
-if [ "$TRANSLATE" == "1" ]; then
-    echo "Running validator for translated files..."
-    CLEAN_CP=1
-
-    cd $CP_HOME
-    tee /var/log/candlepin/translation.log < /tmp/teepipe &
-    validate_translation > /tmp/teepipe 2>&1
-fi
-
-if [ "$SONARQUBE" == "1" ]; then
-    echo "About to upload code to SonarQube server..."
-    CLEAN_CP=1
-
-    SONAR_HOST_URL=$1 # SONARQUBE Host URL
-    BRANCH_NAME=$2    # The branch to analyze
-    CHANGE_ID=$3      # If PR needs to be uploaded, accept the pull request ID
-    CHANGE_TARGET=$4  # If PR needs to be uploaded, accept the pull request target branch
-
-    tee /var/log/candlepin/sonarqube_upload.log < /tmp/teepipe &
-    sonarqube_upload > /tmp/teepipe 2>&1
-fi
-
-if [ "$UNITTEST" -gt 0 ]; then
-    echo "Running unit tests $UNITTEST time(s)"
-    CLEAN_CP=1
-
-    UTCMD="test"
-    cd $CP_HOME
-    if [ -n "${UNITTEST_FILTER}" ]; then
-        UTCMD="${UTCMD}:${UNITTEST_FILTER}"
-    fi
-
-    # run $UNITTEST time(s) to increase chance of capturing
-    # non-deterministic unit test failures.
-    for (( i=1; i<=$UNITTEST; i++ ))
-    do
-        tee /var/log/candlepin/unit_tests.log < /tmp/teepipe &
-        build_unittest $UTCMD > /tmp/teepipe 2>&1
-        rm -f /tmp/teepipe
-        mkfifo /tmp/teepipe
-    done
-fi
-
 if [ "$DEPLOY" == "1" ]; then
     echo "Deploying candlepin..."
     CLEAN_CP=1
 
     DEPLOY_FLAGS="-g"
 
-    if [ "$SPEC_TESTS" == "1" ] && [ "$HOSTED" == "1" ] \
-        && [ "$CP_NEW_STRUCTURE" == false ] && ($PROJECT_DIR/bin/deploy '-?' | grep -q -- '-H'); then
-        DEPLOY_FLAGS="$DEPLOY_FLAGS -H -a"
-    elif [ "$SPEC_TESTS" == "1" ] && [ "$HOSTED" == "1" ] \
-        && [ "$CP_NEW_STRUCTURE" == true ] && ($PROJECT_DIR/bin/deployment/deploy '-?' | grep -q -- '-H'); then
+    if [ "$HOSTED" == "1" ] && ($PROJECT_DIR/bin/deployment/deploy '-?' | grep -q -- '-H'); then
```

Comment:
I don't think this is working properly. Looking at the Jenkins logs for this PR we can see that the message `This deployment is not for HOSTED` is displayed in the `sh jenkins/spec-test.sh` step for hosted spec tests. Also, if you look at the spec tests that are being run, you can see that only the standalone tests are being run because Candlepin is set to standalone mode.

Reply:
Oh, i missed that. done